### DISCUSSION TITLE
Cooja redirect output

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2002] [Q&A] [answered]
It is amazing to know that Cooja is part of the Contiki toolset. I often use it for clearly understanding what is going on in code. However, when my code fails because of a pointer exception, naturally Cooja would abruptly abort before I would have a chance to see the logs in the Mote output window. Is there a way to redirect Cooja logs to a file for example?

**Comment by nfi** [comment, marked as answer]
In the Mote Output Window, you can select a log file via the menu option `File->Append to file`. The mote output will then be appended to this log file when running the simulation. This option will also be included when saving the simulation configuration.

<img width="337" alt="image" src="https://user-images.githubusercontent.com/605024/176970411-9b3c8d42-b396-460a-a67b-8b21cb03501d.png">

**Reply by rahav** [reply]
Thank you SO much.
